playbooks/google.yml

```diff
@@ -0,0 +1,158 @@
+---
+- name: Provision the GCE Server
+# =======================================
+  hosts: localhost
+  connection: local
+  gather_facts: yes
+
+  vars:
+    zones:
+      "1": "us-central1-a"
+      "2": "us-central1-b"
+      "3": "us-central1-c"
+      "4": "us-central1-f"
+      "5": "europe-west1-b"
+      "6": "europe-west1-c"
+      "7": "europe-west1-d"
+      "8": "asia-east1-a"
+      "9": "asia-east1-b"
+      "10": "asia-east1-c"
+
+  # This variable file is included so the gce-network role knows
+  # which port to open for SSH
+  vars_files:
+    - roles/ssh/defaults/main.yml
+
+  vars_prompt:
+    - name: "gce_zone_var"
+      prompt: >
+        What zone should the server be located in?
```

Comment:
Small thing, but can we make all the parentheses line up? We also only need four spaces after "United States" in order to be consistent with the Amazon configuration. This should make the output look a little cleaner in the terminal.